docker_hub/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM alpine:latest
+ARG ADAPT_VERSION
+RUN apk --update add --no-cache nodejs npm openssh \
+  && rm -rf /var/cache/apk/*
+RUN npm install -g @adpt/cli@$ADAPT_VERSION
```

Comment:
You'll also need yarn. The apk version looks a bit old, so add yarn@~1.19 to the npm install.

Reply:
Done.